### ISSUE TITLE
Source data issue: "CCSDS 701.00-R-3 (Red Book)” should just be "CCSDS 701.00-R-3”

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-data-ccsds#8]
https://public.ccsds.org/Publications/SilverBooks.aspx

This document is a Red Book that is later published into a Blue Book. The identifier here is an anomaly because it incorporates the text “ (Red Book)”. Technically, it should only be ""CCSDS 701.00-R-3” as the “R” already says it is a Red Book.

<img width="831" alt="image" src="https://github.com/relaton/relaton-data-ccsds/assets/11865/43fb0a58-5635-4513-90f7-4c7592e0bda2">

In the actual linked publication: https://public.ccsds.org/Pubs/701RB3.pdf , it clearly says the identifier is just “CCSDS 701.00-R-3”.

<img width="295" alt="image" src="https://github.com/relaton/relaton-data-ccsds/assets/11865/b2f8b018-ad1d-41e8-93d4-347688e31ec4">

Proposed action: change identifier "CCSDS 701.00-R-3 (Red Book)” into "CCSDS 701.00-R-3”.

**Comment by andrew2net** [comment]
@ronaldtse the type is "Silver Book". Shouldn't the ID be "CCSDS 701.00-R-3-S"?

**Comment by ronaldtse** [comment]
@andrew2net yes you're right. This is now a silver book so it should be "CCSDS 701.00-R-3-S".

**Comment by ronaldtse** [comment]
Reported to CCSDS.

**Comment by ronaldtse** [comment]
CCSDS reported this fixed. Thanks!

<img width="781" alt="image" src="https://github.com/relaton/relaton-data-ccsds/assets/11865/a37917b1-1fd4-41d9-a72c-09f9e6329202">

Ping @mico @andrew2net .

**Comment by andrew2net** [comment]
> @andrew2net yes you're right. This is now a silver book so it should be "CCSDS 701.00-R-3-S".

@ronaldtse it should be "CCSDS 701.00-R-3-S" but it's "CCSDS 701.00-R-3" now.

**Comment by ronaldtse** [comment]
@andrew2net reported:

## CCSDS 701.00-R-3 is a Silver Book

https://public.ccsds.org/Publications/SilverBooks.aspx

On the Silver Books page, the “CCSDS 701.00-R-3” entry was corrected (the “(Red Book)” suffix was removed), but is now missing the “-S” suffix at the end since it is now a Silver Book.

Proposed action: Rename "CCSDS 701.00-R-3” => "CCSDS 701.00-R-3-S"
